Source/Core/DolphinQt/MenuBar.cpp

```diff
@@ -1123,6 +1134,37 @@ void MenuBar::UpdateToolsMenu(const Core::State state)
   }
 }
 
+#ifdef RC_CLIENT_SUPPORTS_RAINTEGRATION
+void MenuBar::UpdateToolsAchievements()
```

Comment:
How about naming this something like UpdateAchievementsDevelopment? The current name both has redundant information (Tools) and leaves out the fact that this doesn't affect the "regular" achievements menu.